docs/internals/organization.txt

```diff
@@ -21,106 +21,130 @@ and its community.
+- Requesting a vote of the technical board when a major change reaches one of
+  its implementation milestones and is intended to merge.
 
-Prerogatives
-------------
+Membership
+----------
+
+`The technical board`_ select Mergers_ as necessary to maintain their number at
+a minimum of three, in order to spread the workload and avoid over-burdening or
+burning out any individual Merger. There is no upper limit to the number of
+Mergers.
+
+It's not a requirement that a Merger is also a Django Fellow, but the Django
+Software Foundation has the power to use funding of Fellow positions as a way
+to make the role of Merger sustainable.
+
+The following restrictions apply to the role of Merger:
+
+- A person must not simultaneously serve as a member of the technical board. If
+  a Merger is elected to the technical board, they shall cease to be a Merger
+  immediately upon taking up membership in the technical board.
+- A person may serve in the roles of Releaser and Merger simultaneously.
+
+The selection process, when a vacancy occurs or when the technical board deems
+it necessary to select additional persons for such a role, occur as follows:
+
+- Any member in good standing of an appropriate discussion venue, or the Django
+  Software Foundation board acting with the input of the DSF's Fellowship
+  committee, may suggest a person for consideration.
+- The technical board consider the suggestions put forth, and then any member
+  of the technical board formally nominate a candidate for the role.
+- The technical board vote on nominees.
+
+Mergers may resign their role at any time, but should provide some advance
+notice in order to allow the selection of a replacement. Termination of the
+contract of a Django Fellow by the Django Software Foundation temporarily
+suspend that person's Merger role until such time as the technical board can
+vote on their nomination.
+
+Otherwise, a Merger may be removed by:
 
-Core team members may participate in formal votes, typically to nominate new
-team members and to elect the technical board.
+- Becoming disqualified due to election to the technical board.
+- Becoming disqualified due to actions taken by the Code of Conduct committee
+  of the Django Software Foundation.
+- A vote of the technical board.
 
-Some contributions don't require commit access. Depending on the reasons why a
-contributor joins the team, they may or may not have commit permissions to the
-Django code repository.
+.. _releasers-team:
 
-However, should the need arise, any team member may ask for commit access by
-writing to the core team's mailing list. Access will be granted unless the
-person withdraws their request or the technical board vetoes the proposal.
+Releasers
+=========
+
+Role
+----
 
-Core team members who have commit access are referred to as "committers" or
-"core developers".
+Releasers_ is a small set of people who has the authority to upload packaged
```

Comment:
Apologies for noise ;) Same principle as prior, `is a small set of people who has` => `are a small set of people who have`